### DISCUSSION TITLE
Smooth upgrade to new version

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#19284] [Q&A and General discussion] [answered]
**- Is there any wiki about how to Smooth upgrade to new version(in 2.1.x) ?**
Suppose that in the online scenario, What is the order of component upgrade, so that we can ensure the overall stability and minimize the impact on online services

**Comment by yanliang567** [comment, marked as answer]
how did you deploy the Milvus, by helm, operator or any other scripts?

**Reply by bingbb47** [reply]
by helm ,but we reorganized ; We hope to follow a certain order to upgrade

**Reply by yanliang567** [reply]
I think the order is controlled by k8s. @LoveEachDay any ideas?
